### PR TITLE
Minor blog issues fixes, author detiails, comparison component ++

### PR DESCRIPTION
**Pull request by zriyansh**

### CODE REVIEW

File: app/resource-center/OpenTelemetry/OpenTelemetry.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default function OpenTelemetry() {
   const posts = allCoreContent(sortPosts(allOpentelemetries))
 
   const primaryFeaturedBlogs = posts.slice(0, 2)
-  const secondaryFeaturedBlogs = posts.slice(2, 12)
+  const secondaryFeaturedBlogs = posts.slice(2, 200)
```
Comment (YounixM) [inline app/resource-center/OpenTelemetry/OpenTelemetry.tsx:35]: Initially we had set an arbitrary number, but this should be equal to the total number of blogs

File: components/Admonition/Admonition.styles.css [hunk 2]
```diff
@@ -27,23 +27,30 @@
   
   .admonition-note {
     /* background-color: #e7f5ff; */
-    border-color: #80bfff;
+    border-color: #767373;
```
Comment (YounixM) [inline components/Admonition/Admonition.styles.css]: Use css variables , check if tailwind colors has this color

File: components/Admonition/Admonition.styles.css [hunk 3]
```diff
@@ -27,23 +27,30 @@
...
   }
   
   .admonition-tip {
-    background-color: #e8e9e8;
+    background-color: #094903;
```
Comment (YounixM) [inline components/Admonition/Admonition.styles.css]: Same as above

File: components/mainFooter.tsx [hunk 4]
```diff
@@ -15,97 +15,75 @@ function Footer() {
...
-                <Link href="#" className="mt-5">
-                  Privacy Policy
-                </Link>
-                <Link href="#" className="mt-5">
-                  Terms of Service
-                </Link>
+              
                 <div className="mt-5 flex items-center gap-2 pr-7 max-md:pr-5">
-                  <Link href="#">Press Kit</Link>
+                  <Link href="https://signoz.io/slack" target="__blank">Slack</Link>
```
Comment (YounixM) [inline components/mainFooter.tsx:40]: _blank

File: components/mainFooter.tsx [hunk 5]
```diff
@@ -15,97 +15,75 @@ function Footer() {
...
                   <ArrowUpRight size={16} />
                 </div>
                 <div className="mt-5 flex items-center gap-2 whitespace-nowrap pr-8 max-md:pr-5">
-                  <Link href="#">Contact</Link>
+                  <Link href = "https://twitter.com/SigNozHQ" target="__blank">Twitter</Link>
+                  <ArrowUpRight size={16} />
+                </div>
+                <div className="mt-5 flex items-center gap-2 whitespace-nowrap pr-8 max-md:pr-5">
+                  <Link href="https://community-chat.signoz.io/" target="__blank">Community Archive</Link>
```
Comment (YounixM) [inline components/mainFooter.tsx:48]: _blank

File: css/global.css [hunk 6]
```diff
@@ -38,8 +38,10 @@ pre {
 
 .code-line {
   display: block;
-  padding-left: 16px;
-  padding-right: 16px;
+  background-color: rgba(103, 103, 126, 0.323);
+  /* padding-left: 16px; */
```
Comment (YounixM) [inline css/global.css:42]: Remove commented code

File: app/resource-center/OpenTelemetry/OpenTelemetry.tsx (hunk 1 above)
Comment (zriyansh) [reply app/resource-center/OpenTelemetry/OpenTelemetry.tsx]: done

**Review by YounixM — APPROVED** [review]

**Pull request merged**